<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QWidget" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lineas de Procesos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(183, 223, 254);</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>971</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="BttmKia">
          <property name="text">
           <string>Kia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BttmToyota">
          <property name="text">
           <string>Toyota</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BttmAudi">
          <property name="text">
           <string>Audi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BttmBMW">
          <property name="text">
           <string>BMW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BttmFord">
          <property name="text">
           <string>Ford</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BttmNissan">
          <property name="text">
           <string>Nissan</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="linea1"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="linea2"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="linea3"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="linea4"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="linea5"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="linea6"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>447</width>
         <height>577</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>451</width>
          <height>581</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Carros"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
